app/src/main/kotlin/com/wire/android/ui/calling/ongoing/participantsview/ParticipantTile.kt

```diff
@@ -90,10 +92,11 @@ fun ParticipantTile(
     onSelfUserVideoPreviewCreated: (view: View) -> Unit,
     onClearSelfUserVideoPreview: () -> Unit
 ) {
+    val alpha = if (participantTitleState.hasEstablishedAudio) 1f else 0.5f
     Surface(
-        modifier = modifier,
+        modifier = modifier.alpha(alpha),
```

Comment:
I think this also makes the background of the surface to be 0.5 visible and by looking at the designs we just need to make the username text to be greyed out, right?
Because the alpha is set here, also "Connecting..." red text has this alpha applied and looks way darker than on designs 😅 
<img width="351" alt="image" src="https://github.com/wireapp/wire-android-reloaded/assets/30429749/20e00a91-4656-4cda-83b6-1d1a709ebbbd">

Reply:
Well,  it's also greyed out, if compare the two tiles bellow you can see the difference

<img width="345" alt="Screenshot 2023-08-17 at 16 10 03" src="https://github.com/wireapp/wire-android-reloaded/assets/18124919/1b3c6fc1-21a8-45b9-9d19-45e211775777">

Maybe I would need to adjust the level of alpha to be as the design spec

Reply:
Or just pass it to design review

Reply:
<img width="191" alt="image" src="https://github.com/wireapp/wire-android-reloaded/assets/9389043/f36b2648-6fae-462c-946c-df8dea4b0c24">


The "connecting" is really hard to read. Even with what I judge to be good eyes and a good screen, it's a bit hard for me. I don't think this is accessible.

Reply:
I've update the alpha to 0.6

Reply:
I don't think the Alpha is the issue here. It's too dark anyway.
Shouldn't it be a lighter variant of the red colour?

Reply:
indeed, alpha is not needed for texts, I will just tak the exact color from Figma and apply the alpha only on the image

Reply:
I think alpha shouldn't be used here - on designs it's not used, and as you can see both connected and connecting have the same exact black color of the background, the only difference is colors of the text:
![image](https://github.com/wireapp/wire-android-reloaded/assets/30429749/2ac7aec7-bb02-4659-a2d3-346fadc59c73)

Reply:
Exactly that's what I just wrote 😄